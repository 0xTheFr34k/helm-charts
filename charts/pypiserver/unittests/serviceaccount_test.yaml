suite: test service account

templates:
  - serviceaccount.yaml

tests:
  - it: should be empty when service account is not created
    set:
      serviceAccount.create: false
    asserts:
      - hasDocuments:
          count: 0

  - it: should match snapshot of default values
    set:
      serviceAccount.annotations:
        my-key: my-value
    release:
      name: pypiserver
      namespace: pypi
    chart:
      version: 1.0.0
      appVersion: "v1.0.0"
    asserts:
      - matchSnapshot: { }
