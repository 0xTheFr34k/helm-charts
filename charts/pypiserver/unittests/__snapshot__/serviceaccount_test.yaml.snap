should match snapshot of default values:
  1: |
    apiVersion: v1
    automountServiceAccountToken: true
    kind: ServiceAccount
    metadata:
      annotations:
        my-key: my-value
      labels:
        app.kubernetes.io/instance: pypiserver
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: pypiserver
        app.kubernetes.io/version: v1.0.0
        helm.sh/chart: pypiserver-1.0.0
      name: pypiserver
